contracts/CertifiedFile.sol

```diff
@@ -4,66 +4,75 @@ pragma solidity <0.6.0;
 Gas to deploy: 655.214
 */
 
-import "./interfaces/UserInterface.sol";
 import "./interfaces/CertifiedFileInterface.sol";
-import "./interfaces/CertifiedFileCheckerInterface.sol";
+import "./interfaces/SignaturitUserInterface.sol";
 
 
 contract CertifiedFile is CertifiedFileInterface {
-    CertifiedFileCheckerInterface public certifiedFileChecker;
-
     address public signaturit;
     address public owner;
 
+    string constant CREATED_EVENT = 'certified_file.contract.created';
+    string constant NOTIFICATION_EVENT = 'certified-file-nofify';
```

Comment:
since this is the key for a list of address to notify, i would call it NOTIFIERS_KEY = 'certified-file-notifiers' giving it more context